jujugui/static/gui/src/app/components/deployment/summary/summary.js

```diff
@@ -348,6 +437,7 @@ YUI.add('deployment-summary', function() {
       return (
         <div className="deployment-panel__child">
           <juju.components.DeploymentPanelContent
+            className="deployment-summary"
```

Comment:
You could just add this class to the above wrapping div rather than adding this extra className functionality to the DeploymentPanelContent component. Might make that component slightly less convoluted in its purpose.